src/MediaRatingsSecondary.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Layered implementations of secondary methods for {@code Map}.
+ *
+ * @param <String>
+ *            String entries for media
+ * @param <Integer>
+ *            Integer values for rating
+ */
+@SuppressWarnings("hiding")
+public abstract class MediaRatingsSecondary<String, Integer>
```

Comment:
I still think the generic types here are unnecessary. Usually, you would specify them as K, V, so the user could specify them. When they're already defined like this, they can probably be omitted.